recipe/meta.yaml

```diff
@@ -22,13 +22,14 @@ requirements:
     - ipykernel
     - traitlets
     - notebook
+    - widgetsnbextension
 
 test:
   imports:
     - ipywidgets
 
 about:
-  home: http://github.com/jakevdp/ipywidgets
```

Comment:
Well, he did have something like that once. ( http://github.com/jakevdp/ipywidgets-static )